README.md

```diff
@@ -1,39 +1,82 @@
-# Get started
+# Bazaar Finance
 
-1. Clone the repo and cd into it `git clone https://github.com/symfoni/hardhat-react-boilerplate.git MyProject && cd MyProject`
-2. Install deps with yarn `yarn` or npm `npm install`
-3. Start hardhat `npx hardhat node --watch`
+Bazaar Finance allows developers to capture value for their work in open-source software though decentralized finance (DeFi) protocols. Thanks to lending protocols, users around the globe now have a new avenue to fund the open-source tools they find valuable. Not by dipping into their wallets, but via interest earned on their savings. Instead of having to actively give lump sums or set up recurring monthly donations, people can allocate a portion of interest they earn, passively and transparently, with the ability to withdraw their principle at any time.
```

Comment:
nice intro! do we wanna prioritize 2 additional product values to differentiate us from projects like rDAI:

1. for recipient: they can have a recurring source of income, predictably
2. for depositor: they can earn additional yield, making it superior to simply just holding the underlying asset w/o depositing with Bazaar